kubernetes/ansible/roles/sunbird-deploy/tasks/main.yml

```diff
@@ -38,13 +38,13 @@
   copy:
     dest: "{{ chart_path }}/keys/{{ adminutil_refresh_token_public_key_kid }}"
     content: "{{ core_vault_sunbird_sso_publickey }}"
-  when: release_name == "learner" or release_name == "lms" or release_name == "groups" or release_name == "knowledgemw"
+  when: release_name == "learner" or release_name == "lms" or release_name == "groups" or release_name == "knowledgemw" or release_name == "ml-survey-service"
```

Comment:
Why do you need to add your service here where you are doing the same copy step in the next block?

Reply:
removed